### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by preciousuwhubetine**
In this branch, I wrote SQL statements to query and update the `animals` table.

### Part 1
- I added a `species` column of type `string` to the table.
- I added 7 new rows to the table.

### Part 2
I started a new transaction and did the following:
- I set the `species` column of all animals to `unspecified`.

I rolled back the transaction.

I began a new transaction and did the following:
- I set the `species` column of all animals whose names end with `mon` to `digimon`.
- I set the `species` column of the remaining animals to `pokemon`.

I commited the transaction.

Output:
transaction1

I began a new transaction and did the following:
- I took a deep breath, then deleted all the records from the table.

I rolled back the transaction.

Output:
transaction2

I began a new transaction and did the following:
- I deleted all animals born after the 1st of January, 2022.

I created a new savepoint for the transaction.
- I updated the weight of all animals by flipping the signs in the `weight_kg` column (multilying by -1).

I rolled back the transaction to the earlier created savepoint.
- I updated the weight of all the animals where the `weight_kg` column had a negative value by multiplying by -1.

I commited the transaction.

Output:
transaction3

### Part 4
I wrote the following SQL queries:
- A query to count how many animals were in the database.
- A query to count how many animals have never tried to escape.
- A query to get the average weight of the animals.
- A query to check the type of animals that tried the most to escape (Neutered or not).
- A query to get the minimum and maximum weight of both classes of animals (Neutered and Non neutered).
- A query to get the average number of times each class of animal (Neutered or not) born between 1900 and 2000 has tried to escape.

Output:
query1
query2

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -9,3 +9,9 @@ SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+SELECT neutered, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY neutered;
+SELECT neutered, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1-1-1990' AND '1-1-2000' GROUP BY neutered;
```
Comment (Zuhaib042) [inline queries.sql:17]: - [x] Well done running the queries but I double-checked that the screenshots you shared in the last part of your PR are not the result of these queries. Although these queries are not wrong and may fulfill the requirements, the results screenshots should be in the PR. kindly consider fixing it.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @preciousbetine 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Implemented most of the requirements
✅ Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (preciousuwhubetine) [reply queries.sql:17]: @Zuhaib042 Oh, my bad! I used some old screenshots. Updating them now...

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @preciousbetine, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**